cfme/tests/intelligence/reports/test_reports.py

```diff
@@ -201,7 +204,8 @@ def test_report_edit_secondary_display_filter(appliance, filter_report, soft_ass
     Bugzilla:
         1565171
     """
-    filter_report.update(
+    report = get_report("filter_report", "test_filter_report")
```

Comment:
Q: why do you want the report to be instantiated during the test? I.e. why have you implemented the wrapper in `get_report`?

Reply:
Ah, now I see, because of the `menu_name` and `filter_name`